custom_components/smartthinq_sensors/sensor.py

```diff
@@ -297,6 +312,78 @@
     DeviceType.WASHER,
 ]
 
+RANGE_SENSORS = {
+    DEFAULT_SENSOR: {
+        ATTR_MEASUREMENT_NAME: "Default",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FN: lambda x: x._power_state,
+        ATTR_ENABLED: True,
+    },
+    FEAT_COOKTOP_STATE: {
```

Comment:
This probably is better to define as `binary_sensor`. It is only on/off and HA history is more readable

Reply:
I was following the pattern used in all the other devices. The default state doesn't seem to be binary for any of the other devices. Should we change this for all devices then?

I can change "Cooktop" and "Oven" to binary sensors, though.

Reply:
Yes, default leave as standard sensor. Just talking about `FEAT_COOKTOP_STATE` and `FEAT_OVEN_STATE`.

Reply:
Done.